### PR TITLE
Update realistic-app-benchmark.md

### PR DESCRIPTION
**Pull request by jnativio**
This is the contents I added for this section. 
I am suggesting to change the name to 'customer-like application'.
I am not confident yet that the included pictures will render correctly, so we will see.

### CODE REVIEW

**Comment by ldemailly** [comment]
if you look at https://deploy-preview-1568--preliminary-istio.netlify.com/docs/performance-and-scalability/realistic-app-benchmark/ the images are squished, probably because you didn't set the aspect ratio (#1460)

File: content/docs/performance-and-scalability/realistic-app-benchmark.md [hunk 1]
```diff
@@ -1,9 +1,84 @@
...
+Three configurations are tested for each build:
+
+**•	Istio Full:** Istio is deployed to the K8s cluster using the manifest file that comes in the build, with the default configuration. Some changes are made to force pods to certain nodes.
+
+**•	No Mixer:** the Telemetry and Policy functions of Mixer are disabled in order to isolate their impact on the overall performance.
+
+**•	Ingress Only:** the application is re-deployed without sidecars, so basically just the Istio Ingress is being exercised. This provides an approximate baseline where Istio is not used.
+
+The regression patrol perform these main tasks:
+1.	Checks for new builds in the [Istio Daily Build Repository](https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/ );
```
Comment (ldemailly) [inline content/docs/performance-and-scalability/realistic-app-benchmark.md]: on the preview this didn't turn into a bullet list

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 13c5b989240c4477b83c922fc82d01004fd68af4

https://app.netlify.com/sites/preliminary-istio/deploys/5b2d7f0bc96592102cecd188

File: content/docs/performance-and-scalability/realistic-app-benchmark.md [hunk 2]
```diff
@@ -1,9 +1,81 @@
...
+
+•	**Auth Service** generates JWT if the user/password is valid.
+
+•	**Main Service** primarily consists of the presentation layer (web pages) that interact with the other services. This allows the user to interact directly with the application via browser, but it is not exercised during the load test.
+
+
+The diagram below represents the different pods/containers of the application in the Kubernetes/Istio environment:
+
+{{< image width="100%" ratio="100%"
+    link="https://ibmcloud-perf.istio.io/regpatrol/istio_regpatrol_readme_files/image004.png
```
Comment (ldemailly) [inline content/docs/performance-and-scalability/realistic-app-benchmark.md]: missing close quote

**Comment by ldemailly** [comment]
you need to add entries to the dictionary file I mentionned for the build to pass

**Comment by jnativio** [comment]
@ldemailly I have updated the dictionary.

**Comment by ldemailly** [comment]
you forgot git push ? (or pull first as I fixed a typo in your fork)

**Comment by jnativio** [comment]
Your commit is already of my patch-1 branch which is in the PR. When I try push from my local git it says it up-to-date. So, I am confused about what is missing. I think I started in wrong way this. Maybe it is better to start over. The problem I am leaving pretty soon. Can this wait until I come back?  If not, could you pls get the content and create a new PR yourself?

File: content/docs/performance-and-scalability/realistic-app-benchmark.md (hunk 1 above)
Comment (sdake) [reply content/docs/performance-and-scalability/realistic-app-benchmark.md]: if a numbered list is desired use
```
1. item one 
1. item two
1. item thre
```

or a bulleted list
```
- item 1
- item 2
- item 3
```

The problem here is probably the extra spaces, coupled with the numbers not always starting with 1.

File: content/docs/performance-and-scalability/realistic-app-benchmark.md [hunk 3]
```diff
@@ -1,9 +1,84 @@
 ---
-title: Realistic Application Benchmark
+title: Customer-like Application Benchmark
 description: Performance measurement through realistic micro service application tests.
 weight: 40
 ---
```
Comment (sdake) [inline content/docs/performance-and-scalability/realistic-app-benchmark.md]: might add

``` 
## Overview
```

File: content/docs/performance-and-scalability/realistic-app-benchmark.md [hunk 4]
```diff
@@ -1,9 +1,84 @@
...
+3.	Injects the sidecars to the application and deploys it; 
+4.	Drives the workload, which consists in a sequence of 5-minute runs, each one using a growing number of clients making http requests. 
+5.	Collect and publish the results.
+6.	Repeat the previous steps for each configuration.
+
+In addition to the data collected by the scripts driving the test, monitoring probes in the worker nodes provide real-time metrics displayed on Grafana dashboards. That includes CPU and memory usage, disk activity, network traffic, and other OS metrics on the worker nodes as well as CPU and memory usage at pod and container levels. There are also dashboards for the Prometheus data collected from Istio Control Plane components. The collected metrics are persisted so these dashboards can be reviewed later for further analysis.
+
+
+
+Environment
```
Comment (sdake) [inline content/docs/performance-and-scalability/realistic-app-benchmark.md]: change syntax to
```
## Environments
```

**Comment by ldemailly** [comment]
your branch (pr) only has 1 file changed, you need to change the dictionary file I mentioned above

**Comment by geeknoid** [comment]
Hi, I did some refactoring and the file you edited has moved in the repo. Please rebase and push again. Sorry for the inconvenience.

**Comment by ldemailly** [comment]
addresses (partially) #1377

**Comment by geeknoid** [comment]
Hi there,

To get this PR through, you will need to sync your tree, resolve any conflicts, and then update the PR. Once you update the PR, it'll run the current set of tests and hopefully they'll pass and then this PR can go in.

**Pull request closed without merging**